activesupport/lib/active_support/core_ext/time/compatibility.rb

```diff
@@ -1,5 +1,12 @@
 require "active_support/core_ext/date_and_time/compatibility"
+require "active_support/core_ext/module/remove_method"
 
 class Time
-  prepend DateAndTime::Compatibility
+  include DateAndTime::Compatibility
```

Comment:
Changes `prepend` to `include`. Not sure about this. Really it only is including the `preserve_timezone` property and the `to_time` we want to override. When it was prepending it would never reach the `to_time` in this file. Open to feedback.